LICENSE

```diff
@@ -0,0 +1,21 @@
+MIT License
+
+Copyright (c) 2021 Marek Bielańczuk, Tait Brown, Leo Pfeifenberger, Grzegorz Russek and Contributors
```

Comment:
Invoking @taitems! 📟 

Since this repo is a (heavily modified) fork, and since the upstream has no license file of its own (though it does specify an MIT license), I wonder what to do here. IANAL, but I guess the copyright line could (should?) also include your name, but not sure. (Including any other contributors' names in the copyright seems strange to me.)

What do you think?

Reply:
After some quick investigation, it seems that both this fork and the upstream were published on GitHub in 2011, so I might still set both of your names, but more like this (cf. [here](https://www.copyrightlaws.com/copyright-symbol-notice-year/#h-which-year-should-you-use-in-your-copyright-notice) and [here](https://www.copyrightlaws.com/copyright-symbol-notice-year/#h-what-about-blogs-and-websites)):

```suggestion
Copyright (c) 2011 Marek Bielańczuk
Copyright (c) 2011–2021 Tait Brown
```
...but again, IANAL.

Reply:
Happy to go with your suggestion here, but I also think given your incredible contributions over the years you should probably be on that list too? I originally included Leo et al owing to their efforts.

Reply:
I appreciate the sentiment! 😄  For copyright stuff, though, I might feel more comfortable leaving credits to the "original authors" and maybe separately adding a CREDITS file to list contributors if you prefer (though GitHub has [the equivalent](https://github.com/taitems/jQuery.Gantt/graphs/contributors) built-in).